server/app/views/admin/programs/ProgramIndexView.java

```diff
@@ -665,4 +708,14 @@ private ButtonTag renderManageProgramAdminsLink(ProgramDefinition program) {
             .withClass(ButtonStyles.CLEAR_WITH_ICON_FOR_DROPDOWN);
     return asRedirectElement(button, adminLink);
   }
+
+  private ButtonTag renderExportProgramLink(ProgramDefinition program) {
+    String adminLink = routes.AdminExportController.index(program.id()).url();
+    ButtonTag button =
+        makeSvgTextButton("Export program", Icons.DOWNLOAD)
+            // maybe don't need this id?
```

Comment:
**suggestion:** Seems like a legacy comment?

Reply:
removed this comment and the id.